lib/common/events.ts

```diff
@@ -24,6 +29,9 @@ export const ZONE_SYMBOL_PREFIX = '__zone_symbol__';
 
 const EVENT_NAME_SYMBOL_REGX = /^__zone_symbol__(\w+)(true|false)$/;
```

Comment:
since you are already creating TRUE_STR and FALSE_STR on line 10 and line 11, why not reuse those consts here by generating the regex with something like
  `let evt_name_sym_regx = new RegExp('^__zone_symbol__(\\w+)(' +TRUE_STR  + '|' + FALSE_STR + ')\$');`